python-checks/src/test/resources/checks/caughtExceptions.py

```diff
@@ -61,3 +61,57 @@ def foo():
     ...
   except smtplib.SMTPServerDisconnected:  # OK, SMTPServerDisconnected when Python version is unknown
     ...
+
+def except_star():
+  try:
+    ...
+  except* CustomException: # Noncompliant {{Change this expression to be a class deriving from BaseException or a tuple of such classes.}}
+    ...
+  except* (None, BaseException):  # Noncompliant
+    ...
+  except* list():  # Noncompliant
+    ...
```

Comment:
I think we don't need to duplicate everything, only one or two test cases are enough